src/sentry/static/sentry/app/components/eventOrGroupExtraDetails.tsx

```diff
@@ -152,4 +185,19 @@ const LoggerAnnotation = styled(AnnotationNoMargin)`
   color: ${p => p.theme.textColor};
 `;
 
+const ShadowlessProjectBadge = styled(ProjectBadge)`
+  * > img {
+    box-shadow: none;
+  }
+`;
+
+const BadgeWrapper = styled('div')`
+  display: flex;
+  justify-content: center;
+
+  & > div {
```

Comment:
I believe you no longer need this if you're using the new tag styles, the old unhandled tag styles had a margin